aiojobs/_scheduler.py

```diff
@@ -64,21 +64,17 @@ def pending_count(self):
     def closed(self):
         return self._closed
 
-    async def spawn(self, coro):
+    async def spawn(self, coro) -> Job:
         # The method is not a coroutine
         # but let's keep it async for sake of future changes
         # Migration from function to coroutine is a pain
-        if self._closed:
-            raise RuntimeError("Scheduling a new job after closing")
-        job = Job(coro, self, self._loop)
-        should_start = (self._limit is None or
-                        self.active_count < self._limit)
-        self._jobs.add(job)
-        if should_start:
-            job._start()
-        else:
-            self._pending.append(job)
-        return job
+        return self.spawn_nowait(coro)
```

Comment:
What's the difference between `await spawn` and `spawn_nowait`? Both would run syncly without yielding execution context to next coroutine.